studio/components/interfaces/Settings/Integrations/IntegrationsSettings.tsx

```diff
@@ -0,0 +1,830 @@
+import { zodResolver } from '@hookform/resolvers/zod'
```

Comment:
one for next time: can we break this down into smaller files? just to make things more readable and maintainable

Reply:
yep